content/en/providers/operations-manuals/howto01_using_igtf_ca_distribution/_index.md

```diff
@@ -0,0 +1,322 @@
+impact, see [EGI document 2745](https://documents.egi.eu/document/2745).
+
+### Notice for sites in EGI that support WLCG
+
+The WLCG collaboration - having so requested EGI and in line with long-term EGI
+developments in AAI - has included a specific CA to its own trust anchor
+distribution. Sites that are also part of WLCG should install BOTH the
+"egi-core" AND "lcg" meta-packages, according to your policies. Note that your
+organisation or NGI may also have a specific policy and may have added or
+removed CAs compared to the EGI core policy. Sites that need compliance with the
+WLCG policy should install BOTH packages, or you will miss out the CERN WLCG
+IOTA CA specific exception see
+[EGI document: Considerations on the coexistence of controlled and flexible community models](https://documents.egi.eu/document/2745)
+for details and the
+[WLCG statement](http://lcg-ca.web.cern.ch/lcg-ca/doc/WLCG-CERN-IOTA-statement-MB.pdf).
+
+The EGI production repository contains all relevant packages for both policies
+(and the WLCG repository contains also all EGI packages). There is no reason to
+reconfigure repository locations. Specifically installing (adding) also the
+`ca-policy-lcg` package may be necessary for system configurations built after
+May 2009.
+
+### Combined Assurance/Adequacy Model
+
+In the first quarter of 2017, full support for differentiated assurance profiles
+(combined assurance/adequacy model, also called "cam") will be introduced in the
+EGI trust fabric infrastructure. This will take the form of an additional trust
+anchor meta-package, and replaces the specific policy mechanism described above.
+Such full support also required _new software and configuration at each resource
+centre_. You must deploy the additional trust anchor meta-packages and the new
+local policies in unison, and never install the _cam_ package without such
+software support.
+
+To enable the combined assurance/adequacy model policy, use:
+
+```shell
+yum install ca-policy-egi-cam
+```
+
+or the equivalent for your distribution or operating system. A tarball
+installation is also available.
+
+#### Implementation notes for trust anchor releases post 1.86
+
+Not installing the new "cam" package does not have any detrimental effect on
+current users - only a new class of users (that can only obtain an opaque
+identifiers and do not do full vetting at their electronic identity provider)
+could be affected, and then only those users that are member of one of the
+communities that has part of the combined-assurance programme: LCG-Atlas,
+LCG-Alice, LCG-LHCb, and LCG-CMS.
+
+Also technically this means that you must **only** install the new
+`ca-policy-egi-cam` packages if you **also** at the _same time_ implement
+VO-specific authorization controls in your software stack. This may require
+reconfiguration or a software update:
+
+- for dCache : full support is expected in release v3.1. Meanwhile, if you
```

Comment:
@dlgroep, dCache 3.1 is quite old, is this part still relevant?

Reply:
The configuration bit is, the version is not - it applies to all current versions.

Reply:
Review comments in-line in the code

Reply:
dropped